CONTRIBUTING.md

```diff
@@ -156,6 +156,21 @@ docker build \
   .
 ```
 
+#### Build on macOS with an Apple Silicon chip
+
+> [!NOTE]
+> When building the code on a macOS with a Silicon chip(e.g. M1-M4), The build can fail with protoc errors.
```

Comment:
🔧 This comment will go out-of-date as soon as M5 comes out. We can avoid that by being less specific.

```suggestion
> [!NOTE]
> When building the code on a macOS with a Silicon chip (e.g. M1, M2, etc), the build can fail with protoc errors.
```